docs/source/python_api/mlflow.metrics.rst

```diff
@@ -86,14 +86,67 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
 
 .. autofunction:: mlflow.metrics.rougeLsum
 
-.. autofunction:: mlflow.metrics.precision_at_k
-
 .. autofunction:: mlflow.metrics.toxicity
 
 .. autofunction:: mlflow.metrics.token_count
 
 .. autofunction:: mlflow.metrics.latency
 
+Retriever Metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following metrics are built-in metrics for the ``'retriever'`` model type, meaning they will be 
+automatically calculated with a default ``retriever_k`` value of 3. 
+
+To evaluate document retrieval models, it is recommended to use a dataset with the following 
+columns:
+
+- Input queries
+- Retrieved relevant doc IDs
+- Ground-truth doc IDs
+
+Alternatively, you can also provide a function through the ``model`` parameter to represent 
+your retrieval model. The function should take a Pandas DataFrame containing input queries and 
+ground-truth relevant doc IDs, and return a DataFrame with a column of retrieved relevant doc IDs.
+
+A "doc ID" is a string that uniquely identifies a document. All doc IDs should be entered as a 
+list of doc ID strings.
+
```

Comment:
`All doc IDs should be entered as a list of doc ID strings.` is a bit confusing. Can we clarify?